README.md

```diff
@@ -1 +1,39 @@
-# Kaidan
+# Kaidan - Cross plattform xmpp client
+
+Installation
+------------
+
+On Debian do the following:
+
+Create a working directory
+
+ * mkdir src
+ * cd src
+
+Fetch swift source
+
+ * wget https://github.com/swift/swift/archive/swift-3.0.zip
+ * unzip swift-3.0.zip
+ * cd swift-swift-3.0/
+
+Install all dependencies to build swiften
+
+ * ./BuildTools/InstallSwiftDependencies.sh
+ * ./scons Swiften
```

Comment:
The number of jobs is dependend from your own computer. I think this setting can easily adjusted by each developer themself.